db/pebble/db.go

```diff
@@ -49,6 +50,15 @@ func NewWithOptions(path string, cacheSizeMB uint, maxOpenFiles int, colouredLog
 		Logger:       dbLog,
 		Cache:        pebble.NewCache(int64(cacheSizeMB * utils.Megabyte)),
 		MaxOpenFiles: maxOpenFiles,
+		Levels: []pebble.LevelOptions{
+			{TargetFileSize: 2 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 4 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 8 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 16 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 32 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 64 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
+			{TargetFileSize: 128 * 1024 * 1024, FilterPolicy: bloom.FilterPolicy(10)},
```

Comment:
if we decide to merge this PR please use `utils.Megabyte` constants

Reply:
Btw, do you understand why for all these levels there is different targetFileSize and same FilterPolicy ?